### ISSUE TITLE
Docker build errors ?

### ISSUE THREAD

**Issue by centminmod** [litespeedtech/lsquic#77]
On CentOS 7,6 64bit docker, the docker build command reports Cmake errors for ` Cannot find source file: ls-qpack/lsqpack.c`

```
docker build -t lsquic .
```
```
Step 9/10 : RUN cd /src/lsquic-client &&     cmake -DBORINGSSL_INCLUDE=/src/boringssl/include           -DBORINGSSL_LIB=/usr/local/lib . &&     make
 ---> Running in a16727bf0c47
-- The C compiler identification is GNU 5.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Build type: Debug
-- Performing Test HAS_NO_IMPLICIT_FALLTHROUGH
-- Performing Test HAS_NO_IMPLICIT_FALLTHROUGH - Success
-- AddressSanitizer is OFF
-- Compiler flags:    -Wall -Wextra -Wno-unused-parameter -fno-omit-frame-pointer -Wno-implicit-fallthrough -Wno-missing-field-initializers -O0 -g3 -Werror 
-- BoringSSL include directory /src/boringssl/include
-- Found /usr/local/lib library: /usr/local/lib/libssl.a
-- Found /usr/local/lib library: /usr/local/lib/libcrypto.a
-- /usr/local/lib library not found
-- Found zlib: /usr/lib/x86_64-linux-gnu/libz.a
-- Found event: /usr/lib/x86_64-linux-gnu/libevent.a
-- Looking for sendmmsg
-- Looking for sendmmsg - found
-- Looking for recvmmsg
-- Looking for recvmmsg - found
-- Looking for open_memstream
-- Looking for open_memstream - found
-- Looking for IP_MTU_DISCOVER
-- Looking for IP_MTU_DISCOVER - found
-- Looking for IP_DONTFRAG
-- Looking for IP_DONTFRAG - not found
-- Looking for include file regex.h
-- Looking for include file regex.h - found
-- Configuring done
CMake Error at src/liblsquic/CMakeLists.txt:108 (add_library):
  Cannot find source file:

    ls-qpack/lsqpack.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx


CMake Error: Cannot determine link language for target "lsquic".
CMake Error: CMake can not determine linker language for target: lsquic
-- Generating done
-- Build files have been written to: /src/lsquic-client
The command '/bin/sh -c cd /src/lsquic-client &&     cmake -DBORINGSSL_INCLUDE=/src/boringssl/include           -DBORINGSSL_LIB=/usr/local/lib . &&     make' returned a non-zero code: 1
```

```
docker info
Client:
 Debug Mode: false

Server:
 Containers: 4
  Running: 1
  Paused: 0
  Stopped: 3
 Images: 14
 Server Version: 19.03.2
 Storage Driver: overlay
  Backing Filesystem: extfs
  Supports d_type: true
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
```

**Comment by dtikhonov** [comment]
Docker instructions and the Docker file have been fixed in 2718741832537e92c6c9824e6a8f1858fe289d22.

**Comment by litespeedtech** [comment]
Fixed -- closing.
